core/lib/dal/src/proof_generation_dal.rs

```diff
@@ -1,4 +1,3 @@
-#![doc = include_str!("../doc/ProofGenerationDal.md")]
```

Comment:
Why did you remove it?

Reply:
SQL schema redesign where we don't need db to be driven by `status` enum anymore. Instead, we can rely on nullable `proved_at`. See related: https://github.com/matter-labs/zksync-era/pull/2474#discussion_r1696601145 I'm open to any suggestions in how to redesign the SQL schema.

Reply:
I added it back. 🫡